src/server.rs

```diff
@@ -29,6 +29,19 @@ async fn index(config: web::Data<RwLock<Config>>) -> Result<HttpResponse, Error>
         .landing_page_content_type
         .clone()
         .unwrap_or("text/plain; charset=utf-8".to_string());
+
+    match &config.server.landing_page_file {
+        Some(file) => {
```

Comment:
This can simply be replaced with a `if let` statement.

Reply:
I am rather new to Rust. No idea how to use a `if let` statement instead. Some of he constructs are still alien to me.
I'll have to read up on that.

Reply:
No worries, it can be refactored as:

```rs
    if let Some(file) = &config.server.landing_page_file {
        let file_contents = fs::read_to_string(file).unwrap_or("".to_string());
        if !file_contents.is_empty() {
            return Ok(HttpResponse::Ok()
                .content_type(content_type)
                .body(file_contents));
        }
    }

```

So that you will get rid of `None => ()` 😃

Reply:
Ah, I see. Thanks.